regression/validate-trace-xml-schema/check.sh

```diff
@@ -13,6 +13,7 @@ fi
 xmllint="$(command -v xmllint)"
 if [ $? -ne 0 ] > /dev/null; then
     echo "xmllint not found, skipping XSD tests"
+    exit
```

Comment:
It seems to me that this should exit with an error code, as we haven't tested successfully if `xmllint` is missing.

Reply:
I'm a fan of this idea - even though it looks to me like we have checked if `xmllint` was found, which is what we care for here, isn't it?